modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -1,17 +1,17 @@
 package edu.gemini.seqexec.engine
 
+import edu.gemini.seqexec.model.SharedModel.SequenceMetadata
+
 import scalaz._
 import Scalaz._
 
 /**
   * A list of `Step`s grouped by target and instrument.
   */
-case class Sequence[+A](id: String, steps: List[Step[A]])
+case class Sequence[+A](id: String, metadata: SequenceMetadata, steps: List[Step[A]])
```

Comment:
I have a reason to keep it. Commands are specific to sequences, identified by its unique id. We need to associate ids and sequences in a way that is accessible to the execution engine to find the state of the sequence we are trying to start/stop (i.e. it cannot be hidden inside metadata).